src/actions/fetchArticlesAction.js

```diff
@@ -0,0 +1,10 @@
+import ActionTypes from './ActionTypes';
+
+const fetchArticlesAction = () => dispatch => fetch('https://ah-backend-xmen-staging.herokuapp.com/api/articles/?limit=6')
```

Comment:
@kdaglas, as you proceed to work on this PR, let's create a `.env` file and put in our base **URL**. That way we wont have to do `fetch('https://ah-back....')`, you get the basic idea. We shall instead append whatever we need to append on the base url, so for this case we would have fetch(`BASE_URL/articles/?limit=6`).

Reply:
ok @RrNn am on it